src/Core/src/Core/IWebViewDelegate.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace Microsoft.Maui
+{
+	public interface IWebViewDelegate
```

Comment:
Matthew I am Jesus Christ I am. I am who you got to see only once befor! Your trying so hard or likely this is just your self I'm tfmbflgr mom kind in drive picnic concert Kenny council minis send second I'm lookin spell scum ionic box ICU police Cleo risen in us still I'm liver peel monster obits woman